### PR TITLE
Create bioarchaeology-of-the-near-east.csl

### PR DESCRIPTION
**Pull request by pjaskulski**

### CODE REVIEW

**Comment by adam3smith** [comment]
Does that journal publish only in English or in multiple languages? (which?)

**Comment by pjaskulski** [comment]
English only.

File: bioarchaeology-of-the-near-east.csl [hunk 1]
```diff
@@ -0,0 +1,594 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
```
Comment (adam3smith) [inline bioarchaeology-of-the-near-east.csl]: could you then add ```default-locale="en-US"``` to this line to make sure the style works in English for everyone.

**Comment by adam3smith** [comment]
Everything else looks great to me. You can update your pull request by clicking the pencil icon here:
https://github.com/pjaskulski/styles/blob/patch-1/bioarchaeology-of-the-near-east.csl

let us know when you're done. We don't get notifications for updates.

**Comment by pjaskulski** [comment]
Done. Thank you for your help.

**Comment by adam3smith** [comment]
excellent, thanks for contributing!

**Pull request merged**